---
source: tests/test_runner.rs
expression: stripped_actual_output.trim()
---
============
INPUT
============
let foo =
  | 0 => 0
  | 1 => 1
  | 1 => 1
  | _ => 2

============
EXIT CODE
============
1

============
STDOUT
============


============
STDERR
============
error:
  ┌─ tests/compiler/typecheck/unreachable_case_integer_1.kk:4:5
  │
4 │   | 1 => 1
  │     ^ Unreachable case
  │
  = This case is unreachable because all possible cases are already handled by previous branches.
